set.mm

```diff
@@ -3165,12 +3165,12 @@ intermediate steps that are essentially incomprehensible to humans (other
   $}
 
   ${
-    syl5rbbr.1 $e |- ( ps <-> ph ) $.
```

Comment:
non-blocking: We usually let the variables appear in (Greek) alphabetic order.  This helps when looking for a theorem doing text search in the file.  I do this occasionally.

Reply:
Hmm, interesting. I'm not sure I'm committed to alphabetic order as such, but I do like the idea of having the names similar to https://us.metamath.org/mpeuni/bitr3d.html which would argue for

```
    bitr3di.1 $e |- ( ph -> ( ps <-> ch ) ) $.
    bitr3di.2 $e |- ( ps <-> th ) $.
    bitr3di $p |- ( ph -> ( ch <-> th ) ) $=
```

Sometime (probably tomorrow) I'll take a look at whether this would be easy or hard to change.